lightly/api/api_workflow_compute_worker.py

```diff
@@ -296,11 +297,25 @@ def schedule_compute_worker_run(
         """
         if runs_on is None:
             runs_on = []
-        config_id = self.create_compute_worker_config(
-            worker_config=worker_config,
-            lightly_config=lightly_config,
-            selection_config=selection_config,
-        )
+        try:
+            config_id = self.create_compute_worker_config(
+                worker_config=worker_config,
+                lightly_config=lightly_config,
+                selection_config=selection_config,
+            )
+        except ApiException as e:
+            eb = json.loads(e.body)
+            eb_code = eb.get("code")
+            eb_error = eb.get("error")
```

Comment:
I had a look through v2config.services.ts and v3config.services.ts - in all cases I can see, we have an APIError with a code and a fairly well formed human readable message, without stringified JSON objects dumped there. That message becomes `eb_error` here.

But... I've included nicer JSON parsing of `eb_error` in case we do this in future.